Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -538,7 +538,7 @@ class KalmanFitter {
       navigator.resetState(
           state.navigation, state.geoContext, stepper.position(state.stepping),
           state.options.direction * stepper.direction(state.stepping),
-          &st.referenceSurface(), targetSurface);
+          &st.referenceSurface(), nullptr);
```

Comment:
Just a suggestion: Maybe add a comment, that no target must be set here because there is a custom aborter, that stops on `result.finished` or something like that.